### PR TITLE
Created design for login page

### PR DESCRIPTION
**Pull request by lasgra**
image
Monkey following text

image
covers eyes on password input

image
Password strong meter on register

### CODE REVIEW

File: src/app/views/login/login.component.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, ElementRef, ViewChild } from '@angular/core';
```
Comment (lasgra) [inline src/app/views/login/login.component.ts]: Remove unused ViewChild!

File: src/app/views/login/login.component.ts [hunk 2]
```diff
@@ -9,4 +9,118 @@ import { Component } from '@angular/core';
 })
 export class LoginComponent {
 
+  emailValidatior = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/
+  mainParts! : ElementRef<any>[]
```
Comment (lasgra) [inline src/app/views/login/login.component.ts]: You don't even use this list 💀💀💀💀

File: src/app/views/login/login.component.ts [hunk 3]
```diff
@@ -9,4 +9,118 @@ import { Component } from '@angular/core';
...
+      this.rightEyePosition.left = `${225 + text.length * 0.3}px`
+    }
+  }
+
+  changeType() {
+    this.resetEyePosition()
+    this.spinAnimation = true
+    setTimeout(() => {
+      this.loginInputIndex = this.loginInputIndex == "1" ? "0" : "1"
+      this.registerInputIndex = this.registerInputIndex == "1" ? "0" : "1"
```
Comment (lasgra) [inline src/app/views/login/login.component.ts:37]: Very smart

**Comment by Majkoo** [comment]
nice

**Comment by lasgra** [comment]
image
Stupid monkey follows mouse when not inputing values

**Comment by lasgra** [comment]
image
new visuals

**Comment by Majkoo** [comment]
koks

**Pull request closed without merging**